logistics/scripts/modules/packaging/nuget-helper.psm1

```diff
@@ -25,6 +28,16 @@ function Install-NuGetCli {
         [string] $sourceNuGetExe = "https://dist.nuget.org/win-x86-commandline/v5.3.1/nuget.exe"
     )
 
+    if (!(Get-IsWindows)) {
+        try {
+            mono -V
+        }
+        catch {
+            Write-Host "ERROR:  Mono is not installed on this Unix system." -ForegroundColor Red
+            return
```

Comment:
In this case, I think we want the process to stop: there is no point in continuing. Therefore I would change this to throw an exception. Before doing so, I would like to get a second opinion from someone on the Platform team when they review this PR. Also changing to "Unix-like" since almost no one actually uses Unix systems anymore 😉.

```suggestion
            throw "ERROR:  Mono is not installed on this Unix-like system."
```

Reply:
Ah yeah, my mistake, the throw just slipped my mind. fixed